### PR TITLE
added page again

### PR DESCRIPTION
**Pull request by stormrider41**
Adding my page

### CODE REVIEW

File: _pages/stormrider41.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+  githubHandle: stormrider41
+  pageURL: stormrider41.github.io/stormrider41/
+  timestamp: TODAYS-DATE-2017/04/25
```
Comment (brianamarie) [inline _pages/stormrider41.yaml]: @stormrider41 You're getting closer! The format should be `yyyy-mm-dd`, like `2017-04-25`.

File: _pages/stormrider41.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+  githubHandle: stormrider41
+  pageURL: stormrider41.github.io/stormrider41/
```
Comment (brianamarie) [inline _pages/stormrider41.yaml]: There are some extra spaces before each word on lines 2, 3, and 4. Make sure that those are gone so the tests will pass. 😄

File: _pages/stormrider41.yaml (hunk 1 above)
Comment (stormrider41) [inline _pages/stormrider41.yaml]: Thank you, but it still does not like it.

**Comment by brianamarie** [comment]
@stormrider41 I went ahead and made some syntax changes - let's see if that works :crossed_fingers:

**Pull request merged**